### PR TITLE
General updates to Rd documentation

### PR DESCRIPTION
**Pull request by kylebaron**
Reformatting all documentation.

- argument descriptions start with lower case and end with period.
- switch to `@md` where we make changes to the other parts of the help topic
- change `##'` to `#'`
- Add examples and return values where we can
- Definitely some files I left alone

Latest pkgdown site:[DOCS.zip](https://github.com/user-attachments/files/17337395/DOCS.zip) Might aid review.

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 template:
   bootstrap: 5
 title: mrgsolve
+url: https://github.com/metrumresearchgroup/mrgsolve
```
Comment (kyleam) [inline _pkgdown.yml:4]: Hmm, isn't this supposed to be the URL of the pkgdown site? At least for [auto-linking](https://pkgdown.r-lib.org/articles/linking.html#across-packages) to work across packages, I think that the value in `_pkgdown.yml` should be `https://mrgsolve.org/docs` and then in `DESCRIPTION` the `URL` field should be

```
URL: https://mrgsolve.org/docs, https://github.com/metrumresearchgroup/mrgsolve
```

File: R/chain.R [hunk 2]
```diff
@@ -16,34 +16,35 @@
...
+#' @param ... unquoted names of compartments or tabled items.
+#' 
+#' @details
+#' There is also a `Req` argument to [mrgsim()] that can 
+#' be set to accomplish the same thing as a call to `Req` in 
+#' the pipeline.
+#' 
+#' Note the difference between `req` and `Req`: the former only 
+#' selects compartments to appear in output while the latter selects both 
+#' compartments and captured items.  Also, when there are items are explicitly
```
Comment (kyleam) [inline R/chain.R:35]: Not new to this PR, but should "when there are items are explicitly listed &#x2026;" be "when there are items explicitly listed &#x2026;" (i.e. drop second "are")?

File: R/chain.R [hunk 3]
```diff
@@ -52,85 +53,95 @@ Req.mrgmod <- function(x,...) {
...
-##' 
-##' @export
+#' Collect only observation records in the simulated output
+#'
+#' @param x model object.
+#' @param value use `TRUE` to collect and return observation records only.
+#' @param ... not used.
+#' 
+#' @details
+#' There is also a `obsonly` argument to [mrgsim()] that can
```
Comment (kyleam) [inline R/chain.R:140]: ﻿typo (not new): "a" -> "an" ?

File: R/class_tgrid.R [hunk 4]
```diff
@@ -107,41 +108,42 @@ as_deslist <- function(data, descol="ID") {
...
-##'
-##' plot(out,CP~., type='b')
-##' }
+#' Create a simtime object
+#'
+#' simtime objects allow the user to specify simulation start and end times, 
+#' along with the simulation time step.
+#'
+#' @rdname tgrid
+#' @name tgrid
```
Comment (kyleam) [inline R/class_tgrid.R:117]: [ not changed with this PR ]

This is one of several spots that use both `@rdname` and `@name` with the same value. In the case, I think `rdname` is unnecessary.

File: R/data_set.R [hunk 5]
```diff
@@ -428,55 +438,57 @@ ev_assign <- function(l, idata, evgroup, join = FALSE) {
...
+#' or every other day (however you want to define that) etc.
+#' 
+#' @param ev an event object.
+#' @param days comma- or space-separated character string of valid days of the
+#' the week (see details).
+#' @param addl additional doses to administer.
+#' @param ii inter-dose interval; intended use is to keep this at the 
+#' default value.
+#' @param unit time unit; the function can only currently handle hours or days.
+#' @param ... event objects named by one the valid days of the week
```
Comment (kyleam) [inline R/data_set.R:462]: typo (not new to this PR): "by one the" -> "by one of the"

File: R/data_set.R [hunk 6]
```diff
@@ -428,55 +438,57 @@ ev_assign <- function(l, idata, evgroup, join = FALSE) {
...
+#' - `w` for Wednesday
+#' - `th` for Thursday
+#' - `f` for Friday
+#' - `sa` for Saturday
+#' - `s` for Sunday
+#' 
+#' 
+#' The whole purpose of this function is to schedule doses on specific
+#' days of the week, in a repeating weekly schedule.  Please do use caution 
+#' when changing `ii` from it's default value.
```
Comment (kyleam) [inline R/data_set.R:478]: typo (not new): "it's" -> "its"

**Review by kyleam — APPROVED** [review]
The cleanups and extensions look good (and like they took a lot of tedious and careful work).

My inline comments are nearly all about existing typos that I spotted while going through the diff.

> argument descriptions start with lower case and end with period.

Great, I think it's very nice to have consistency within a project.

When there's not a clear project preference (our projects in general are all over the place), I tend to start with an upper case letter to align with the [tidyverse convention](https://style.tidyverse.org/documentation.html#documenting-parameters):

> For most tags, like `@param`, `@seealso` and `@return`, the text should be a sentence, starting with a capital letter and ending with a full stop.

File: R/chain.R (hunk 2 above)
Comment (kylebaron) [reply R/chain.R:35]: Right.

File: R/class_tgrid.R (hunk 4 above)
Comment (kylebaron) [reply R/class_tgrid.R:117]: I think they can both go?  this documentation is associated with a function called `tgrid()`.

File: _pkgdown.yml (hunk 1 above)
Comment (kylebaron) [reply _pkgdown.yml:4]: I've been hesitant to list those urls in the package b/c I've been maintaining it and I know CRAN will complain if the url isn't reachable. Well, it's been stable for a while so will just go for it.

**Pull request merged**